grade_test.go

```diff
@@ -184,30 +187,31 @@ func TestCompile(t *testing.T) {
 
 func TestCompileWallSuccess(t *testing.T) {
 	cpp := `#include <iostream>
-	int main() { std::cout << "Hello world!" << std::endl; }`
+	int main() { std::cout << "Hello world!" << std::endl;
+	return 0; }`
 
 	tmp, _ := os.CreateTemp("", "*.cpp")
 	// Removes the created temp file, this will always run
 	defer os.Remove(tmp.Name())
-
+	
 	tmp.WriteString(cpp)
 
 	parts := strings.Split(tmp.Name(), "/")
 	parts = parts[:len(parts)-1]
 
 	dir := strings.Join(parts, "/")
-
-	result := compile(dir, true)
+	result := compile(dir, "c++", true)
 	if !result {
 		t.Error("Compile returned false, expected true.")
 	}
-
+	
 	defer os.Remove(path.Join(dir, "a.out"))
-
+	
 	_, err := os.Open(path.Join(dir, "a.out"))
 	if err != nil {
 		t.Errorf("Failed to open the compiled file: [err=%e]", err)
 	}
+	fmt.Println(result)
```

Comment:
test should not use fmt to print

Reply:
use t methods

Reply:
got rid of the improper usages of the fmt library